packages/eventual-send/src/local.js

```diff
@@ -0,0 +1,39 @@
+const q = JSON.stringify;
+
+const ntypeof = specimen => (specimen === null ? 'null' : typeof specimen);
+
+export const localApplyFunction = (t, args) => {
+  if (!(t instanceof Function)) {
+    const ftype = ntypeof(t);
+    throw TypeError(
+      `Cannot invoke target as a function; typeof target is ${q(ftype)}`,
+    );
+  }
+  return t(...args);
+};
+
+export const localApplyMethod = (t, method, args) => {
+  if (method === undefined || method === null) {
+    return localApplyFunction(t, args);
+  }
+  if (t === undefined || t === null) {
+    const ftype = ntypeof(t);
+    throw TypeError(
+      `Cannot deliver ${q(method)} to target; typeof target is ${q(ftype)}`,
+    );
+  }
+  const fn = t[method];
+  if (!(fn instanceof Function)) {
+    const ftype = ntypeof(fn);
+    if (ftype === 'undefined') {
+      const names = Object.getOwnPropertyNames(t).sort();
```

Comment:
Must include symbol-named properties. Perhaps
```suggestion
      const names = Reflect.ownKeys(t).map(String).sort();
```
in which case, `JSON.stringify` would work for the `q` below. But revisit if you change `q` as I suggest above.

Reply:
I've adapted for use with `assert.quote`.